client/console/src/main/java/org/apache/syncope/client/console/wizards/any/PlainAttrs.java

```diff
@@ -112,7 +115,7 @@
 
             @Override
             public WebMarkupContainer getPanel(final String panelId) {
-                return new PlainSchemas(panelId, schemas, attrTOs);
+                return new PlainSchemasSimple(panelId, schemas, attrTOs);
```

Comment:
Why not name `PlainSchemasSimple` as `PlainSchemasOwn` (considering that the other one is named `PlainSchemasMemberships`)?

Reply:
Good point, I'll do that thanks!